ui/components/app/transaction-detail-item/transaction-detail-item.component.js

```diff
@@ -7,16 +7,30 @@ import {
   FONT_WEIGHT,
   TYPOGRAPHY,
 } from '../../../helpers/constants/design-system';
+import Button from '../../ui/button';
 
 export default function TransactionDetailItem({
   detailTitle,
   detailText,
   detailTotal,
   subTitle,
   subText,
+  handleActionClick,
+  actionText,
 }) {
   return (
     <div className="transaction-detail-item">
+      {actionText && (
```

Comment:
Should we also check `handleActionClick` here, so we aren't displaying a button that does nothing?